pkg/kotsadm/main.go

```diff
@@ -1011,31 +1032,6 @@ func ReadDeployOptionsFromCluster(namespace string, clientset *kubernetes.Client
 		deployOptions.SharedPassword = sharedPassword
 	}
 
-	if deployOptions.IncludeMinio {
```

Comment:
wouldn't removing this cause the `S3AccessKey` and `S3SecretKey` to not be set anymore?

Reply:
wow nice catch.  i have no idea why this is removed...